sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -104,7 +110,7 @@ export interface KeyVaultBackupPollerOptions extends OperationOptions {
 export interface KeyVaultBackupResult {
     endTime?: Date;
     folderUri?: string;
-    startTime: Date;
+    startTime?: Date;
```

Comment:
This was incorrectly marked as required when it may be undefined if the job never started (and this is true to the openAPI specification). Given the small usage of this library I propose we accept this as a small breaking change to fix what is a typing bug